Make.bat

```diff
@@ -23,14 +24,24 @@ GOTO :end
 
 :luvi-sigar
 ECHO "Fetching Luvi Sigar"
-CALL lit.exe get-luvi -o luvi-sigar.exe
+reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set ARCH=ia32 || set ARCH=amd64
+PowerShell -NoProfile -ExecutionPolicy unrestricted -Command "Invoke-WebRequest -Uri https://github.com/virgo-agent-toolkit/luvi/releases/download/%LUVI_VERSION%/luvi-sigar-Windows-%ARCH%.exe -OutFile luvi-sigar.exe"
 if %errorlevel% neq 0 goto error
 GOTO :end
 
 :lit
 ECHO "Building lit"
-PowerShell -NoProfile -ExecutionPolicy unrestricted -Command "[Net.ServicePointManager]::SecurityProtocol =  'Tls12'; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/luvit/lit/%LIT_VERSION%/get-lit.ps1'))"
+IF NOT EXIST luvi-sigar.exe CALL Make.bat luvi-sigar
+if %errorlevel% neq 0 goto error
+mkdir build
+cd build
+git clone --recursive https://github.com/luvit/lit.git
+if %errorlevel% neq 0 goto error
+cd lit
+git checkout %LIT_VERSION%
+..\..\luvi-sigar.exe . -- make . ..\..\lit.exe ..\..\luvi-sigar.exe

```

Comment:
it looks like this line is invoking luvi-sigar.exe to build luvi-sigar.exe.  is that correct?

Reply:
I should stick a comment in there because I was confused too when reading the "build from source" section of lit's README. Apparently the `make` subcommand of `lit` takes three args:
1. the source directory/zip
2. the output exe filename
3. the luvi to bundle into the output exe

Reply:
that helps, thanks!